tests/action_test_params.py

```diff
@@ -1318,6 +1318,33 @@
 
 """)
 
+STORE_RETRIEVE_TO_OBJECT_ALREADY_SAMEVALUE = yaml.safe_load("""
+actions:
+- Create-object:
+    Type: Enterprise
+    Actions:
+    - Set-values:
+        name: enterprise1
+    - Create-object:
+        Type: Infrastructure Access Profile
+        Actions:
+            - Set-values:
+                infrastructure_access_profile_name: access1
```

Comment:
Let's keep the tests simple.  Please remove all of the fields that are not relevant.